sidecar/Makefile

```
@@ -14,6 +14,10 @@ all: build push
 build:
 	docker build --tag $(TAG) --file ${root_dir}sidecar/Dockerfile ${root_dir}sidecar/
 
+build-ubi:
+	docker build --tag $(TAG) --build-arg VERSION=${VERSION} --build-arg RELEASE_NUMBER=${RELEASE_NUMBER} \
+	--file ${root_dir}sidecar/Dockerfile.ubi ${root_dir}sidecar/
```

Formatting nitpick for readability

```suggestion
	docker build \
		--tag $(TAG) \
		--build-arg VERSION=${VERSION} \
		--build-arg RELEASE_NUMBER=${RELEASE_NUMBER} \
		--file ${root_dir}sidecar/Dockerfile.ubi ${root_dir}sidecar/
```